examples/experimental/horizontal-pod-autoscaling/README.md

```diff
@@ -0,0 +1,189 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/experimental/horizontal-pod-autoscaling/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Horizontal Pod Autoscaler
+
+This example presents horizontal autoscaling of pods: an experimental feature delivered in version 1.1 of Kubernetes
+which automatically sets the number of pods in a replication controller (or deployment), base on CPU / memory usage.
+
+
+## Prerequisites
+
+This example requires a running Kubernetes cluster in the version 1.1 with experimental API enabled on API server (```--runtime-config=experimental/v1alpha1=true```),
```

Comment:
This is no longer accurate.  See the changes here: https://github.com/kubernetes/kubernetes/pull/15024

There is now just a single flag to enable the feature.